exporters/stdout/stdoutlog/exporter.go

```diff
@@ -0,0 +1,87 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package stdoutlog // import "go.opentelemetry.io/otel/exporters/stdout/stdoutlog"
+
+import (
+	"context"
+	"encoding/json"
+	"sync/atomic"
+	"time"
+
+	"go.opentelemetry.io/otel/sdk/log"
+)
+
+var zeroTime time.Time
+
+var _ log.Exporter = &Exporter{}
+
+// Exporter is an implementation of  that writes spans to stdout.
```

Comment:
How about something like

```suggestion
// Exporter writes JSON-encoded log records to an [io.Writer] ([os.Stdout]) by default).
```